### PR TITLE
Blog links added to homepage and sidebar

### LINKED ISSUE #563: [Blog] Add links to blog around site
(sub issue of #270)

We have the blog accessible at `/blog`, but no way for the user to find it. The blog should be more accessible but also **more discoverable** so that it will attract normal users of the site.

Links could include
- A link on the landing page
  - This could replace one of the three buttons-- `Browse Courses` and `View Grades` link to the same place-- maybe replace one of these?
- Link on the sidebar, similar to the existing ones
  - Would be nice to have some way to call attention to it, maybe a `New` badge or special color or highlight
- Link on the navbar
  - The search bar is very long-- maybe some of that room could be taken by a blog link? This wouldn't be consistent with other links on the site, though.

### PR DESCRIPTION
**Pull request by ajnye**
## Status

- Done

## GitHub Issues addressed

- This PR closes #563 

## What I did

- Added blog links that link to /lobg in the homepage, replacing redundant "View Grades" and added it to sidebar with a NEW! marker

## Screenshots

- Before
- After
Screen Shot 2022-02-07 at 2 49 39 AM
Screen Shot 2022-02-07 at 2 49 48 AM

## Tests

- A brief explanation of tests done/written or how reviewers can test your work
Run PR and look at sidebar and front page

## Questions/Discussions/Notes

-

## Merging the PR

- Who should merge this PR?
  - [ ] I will merge it myself
  - [x] The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - [ ] preserve the history
  - [ ] squash-merge

### CODE REVIEW

**Comment by alex-shen1** [comment]
Haven't reviewed, but RE: discussion today, is the base branch for the PR supposed to be `dev`? Doesn't make sense to have links on the site be deployed if the blog itself isn't ready yet.

**Comment by ajnye** [comment]
> Haven't reviewed, but RE: discussion today, is the base branch for the PR supposed to be `dev`? Doesn't make sense to have links on the site be deployed if the blog itself isn't ready yet.

You're right, changed base to blog.

File: tcf_website/templates/base/sidebar.html [hunk 1]
```diff
@@ -4,6 +4,15 @@
 <div id="sidebar-wrapper" class="border-right pb-0
      d-flex flex-column justify-content-between text-center">
   <ul class="list-group list-group-flush mt-3">
+    <li>
+        <a href="{% url 'blog' %}"
+            class="list-group-item list-group-item-action pb-0
+        {% if request.resolver_match.url_name == 'blog' %}active{% endif %}">
+        <h4><i class="fa fa-comments fa-fw"></i></h4>
+        <p>NEW! Blog</p>
```
Comment (julesfll) [inline tcf_website/templates/base/sidebar.html:12]: For styling the `New` tag, I would check out Bootstrap's badges: https://getbootstrap.com/docs/5.0/components/badge/

See also the way the future [Save Course is styled](https://github.com/thecourseforum/theCourseForum2/blob/1467bd77a3dc7be5e406400fbdb4b5a57e8eddd9/tcf_website/templates/base/sidebar.html#:~:text=%3C!%2D%2D%20temporary%20badge%20%2D%2D%3E,2%22%3ENew%3C/span%3E)
image

File: tcf_website/templates/base/sidebar.html [hunk 2]
```diff
@@ -4,6 +4,15 @@
 <div id="sidebar-wrapper" class="border-right pb-0
      d-flex flex-column justify-content-between text-center">
   <ul class="list-group list-group-flush mt-3">
+    <li>
```
Comment (julesfll) [inline tcf_website/templates/base/sidebar.html:7]: What do you think about the order of the links on the sidebar? I think putting `Blog` between `Review` and `About` might be good

File: tcf_website/templates/landing/landing.html [hunk 3]
```diff
@@ -87,15 +87,14 @@ <h4 class="card-title">Browse Courses</h4>
...
                         <div class="card-body features-icons-item mx-auto mb-2">
                             <div class="features-icons-icon d-flex">
-                                <i class="icon-chart m-auto text-tcf-indigo"></i>
+                                <i class="icon-bubble m-auto text-tcf-indigo"></i>
                             </div>
-                            <h4 class="card-title">View Grades</h4>
-                            <p class="card-text font-weight-light mb-0">Check out actual grade distributions from past
-                                semesters</p>
+                            <h4 class="card-title">View Blog</h4>
+                            <p class="card-text font-weight-light mb-0">Check out what's going on with your favorite classes from our community</p>
```
Comment (julesfll) [inline tcf_website/templates/landing/landing.html:97]:
```suggestion
                            <p class="card-text font-weight-light mb-0">Join our community and check out what's going on with your favorite classes</p>
```
Might want to discuss this copy some more, but here's a suggestion

**Review by julesfll — COMMENTED** [review]
Great work! Things are looking good; I just left a couple of suggestions for changes.

**Pull request merged**